ssl/s3_lib.c

```diff
@@ -108,9 +108,9 @@ static SSL_CIPHER tls13_ciphers[] = {
         SSL_AEAD,
         TLS1_3_VERSION, TLS1_3_VERSION,
         0, 0,
-        SSL_NOT_DEFAULT | SSL_HIGH,
+        SSL_NOT_DEFAULT | SSL_MEDIUM,
         SSL_HANDSHAKE_MAC_SHA256,
-        128,
+        80,
```

Comment:
Should we add a short comment explaining why it is 80 and not the 128 that you might expect, e.g.

````
/* CCM8 uses a short tag, so we have a lower security strength */
````
Or something like that. Ditto for the other instances.

Reply:
Added.